README.md

```diff
@@ -107,6 +107,47 @@ xpub16y4vhpyuj2t84gh2qfe3ydng3wc37yqzxev6gce380fvvg47ye8um3dm3wn5a64gt7l0fh5j6sj
 ```
 </details>
 
+<details>
+  <summary>How to construct a script and calculate its hash</summary>
+
+  Let's say I have the following recovery phrase
+```
+$ recovery-phrase.prv
+nothing heart matrix fly sleep slogan tomato pulse what roof rail since plastic false enlist
+```
+ Now we create root key
+```
+$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley > root.prv
```

Comment:
I think it will look nicer if the filenames are `recovery-phrase.txt` and `root.xprv`, etc.
And also `verKeyHash1` → `verKey1.hash`.

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/76/commits/112721d68284f0b580e4f96868761380e67ffe1f